app/controllers/petitions_controller.rb

```diff
@@ -9,9 +9,11 @@ class PetitionsController < ApplicationController
   before_action :redirect_to_petition_url, if: :moderated?, only: :moderation_info
 
   respond_to :html
+  respond_to :json, only: [:index, :show]
 
   def index
     @petitions = Petition.visible.search(params)
+    @api_params = params.merge(total_pages: @petitions.total_pages)
```

Comment:
I think we can move this down into the API helper since it has access to the `params` object - no need to pollute the controller with it.